packages/web-forms/src/components/RepeatRange.vue

```diff
@@ -15,7 +15,12 @@ const label = computed(() => props.node.currentState.label?.asString);
 		<RepeatInstance v-for="(instance, index) in node.currentState.children" :key="index" :instance="instance" :instance-index="index" @remove="node.removeInstances(index)" />
 
 		<div class="flex justify-content-start flex-wrap">
-			<Button label="Add" rounded outlined class="w-2 btn-add" @click="node.addInstances()" />
+			<Button rounded outlined class="btn-add" @click="node.addInstances()">
+				<span class="p-button-label" data-pc-section="label">
+					<span class="icon-add" />
+					<span class="btn-add-label">Add {{ label }}</span>
```

Comment:
This would improve vertical alignment of the icon/text:

```suggestion
					<span class="flex btn-add-label">Add {{ label }}</span>
```

Reply:
It looks like this: 
![image](https://github.com/user-attachments/assets/99cc5f43-687d-49d0-85ae-1cd78164e720)

Reply:
Weird! I think I added it to the wrong line. Even then, I think something else changed that affects horizontal alignment. Anyway, as of the current commit ([8382ffa](https://github.com/getodk/web-forms/pull/173/commits/8382ffa83987656a4f0bf74af5fd4a5b0664d40a)), this has the same effect as what I meant to suggest:

```diff
-				<span class="p-button-label" data-pc-section="label">
+				<span class="flex justify-content-center p-button-label" data-pc-section="label">
					<span class="icon-add" />
					<span class="btn-add-label">Add {{ label }}</span>
```